src/experimental/ProbabilisticGraphicalModels/bayesian_network.jl

```diff
@@ -161,3 +161,42 @@ function add_edge!(bn::BayesianNetwork{V,T}, from::V, to::V)::Bool where {T,V}
     to_id = bn.names_to_ids[to]
     return Graphs.add_edge!(bn.graph, from_id, to_id)
 end
+
+function evaluate(bn::BayesianNetwork)
+    logp = 0.0
+    evaluation_env = deepcopy(bn.values)
+    stochastic_index = 1
+    deterministic_index = 1
+
+    for (i, varname) in enumerate(bn.names)
+        is_stochastic = bn.is_stochastic[i]
+        if is_stochastic
+            dist_fn = bn.distributions[stochastic_index]
+            parent_vals = parent_values(bn, i)
+            dist = dist_fn(parent_vals...)
+
+            value = get(evaluation_env, varname, nothing)
+            bijector = Bijectors.bijector(dist)
+            value_transformed = Bijectors.transform(bijector, value)
```

Comment:
I am a bit concerned this might be confusing, I only did this to keep some consistency. In the version of `evaluate` that accept a vector of parameter values, the values are already transformed. Just FYI.